packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -179,7 +180,7 @@ class NestedScrollView extends StatefulWidget {
   /// null.
   const NestedScrollView({
     super.key,
-    this.controller,
+    required this.controller,
```

Comment:
This is likely a change we would not be able to land, it would break every existing Flutter app that uses NestedScrollView.